galsim/angle.py

```diff
@@ -173,6 +173,15 @@ def get_angle_unit(unit):
 This could be appropriate before testing for the equality of two angles for example, or
 calculating the difference between them.
 
+If you want to wrap to a different range than [-pi, pi), you can set the `center` argument
+to be the desired center of the the range.  e.g. for return values to fall in [0, 2pi),
+you would call
+
+    >>> theta = theta.wrap(center=math.pi)
+
+@param center   The center point of the wrapped range. [default: 0]
```

Comment:
Could we allow this to also take an Angle object rather than requiring a number in radians?